### ISSUE TITLE
unable to enter VR, the return value of xr.isSessionSupported  is always false

### ISSUE THREAD

**Issue by Sincenir** [aframevr/aframe#5406]
**Description:**
I am using a-painter from the aframe(https://aframe.io/examples/showcase/a-painter/).

I Don't know everyone's VR can run normally. I can't case to enterVR in either chrome or edge. 
Because  the return value of xr.isSessionSupported is always false.

The only effective way is to add --no-sandbox to chrome setartup. which can be used after disabling the sandbox.
At this time, the retrun value of xr.isSessionSupported is true.

I tried all devices , including Pico and HTC Vive, as well as chrome and edge browsers, and the only thing that worked was --no-sandbox.
But this does not meet my needs. My project is a PWA application and needs to be run on edge, but edge --no-sandbox does not take effect. This feature seems to be unique to chrome.

- A-Frame Version: 1.5.0
- Platform / Device: Pico / HTC Vive  | edge / chrome / edge beta / chrome canary
- Reproducible Code Snippet or URL: https://aframe.io/examples/showcase/a-painter/

**Comment by mrxz** [comment]
WebXR support on desktop is sadly very unreliable (see https://github.com/aframevr/aframe/issues/4709). It really only works with a Chromium based browser (with WebXR compiled/enabled) and a properly setup OpenXR runtime. There _should_ be no need for `--no-sandbox`, but in practice this is often required.

Last I checked Edge did work (with the `--no-sandbox` flag). Unless something changed, it pretty much accepts the exact same command line options as any other Chromium based browser.

You could try reinstalling the SteamVR OpenXR runtime, as during the install process it should try and set itself up correctly, though as you can read in the above linked issue, it's hit or miss.

**Comment by dmarcos** [comment]
Yeah WebXR on desktop is unfortunately not a viable target at the moment. Neither Microsoft nor Google have resources dedicated to it. Many quirks and bugs remain unaddressed for a long time.

**Comment by Sincenir** [comment]
> WebXR support on desktop is sadly very unreliable (see #4709). It really only works with a Chromium based browser (with WebXR compiled/enabled) and a properly setup OpenXR runtime. There _should_ be no need for `--no-sandbox`, but in practice this is often required.
> 
> Last I checked Edge did work (with the `--no-sandbox` flag). Unless something changed, it pretty much accepts the exact same command line options as any other Chromium based browser.
> 
> You could try reinstalling the SteamVR OpenXR runtime, as during the install process it should try and set itself up correctly, though as you can read in the above linked issue, it's hit or miss.



Does your edge use --no-sandbox take effect?
In theory, when it takes effect , it should display the same warning prompt as chrome,indicating that "stability and security have been reduced".
![image](https://github.com/aframevr/aframe/assets/41592290/b43d9c90-35ab-4fe6-92c7-5996659dc568)

But my edge here does not take effect, and no matter what device I connect, I cannot enter VR mode.

Valid only if chrome adds --no-sandbox.

**Comment by Sincenir** [comment]
> Yeah WebXR on desktop is unfortunately not a viable target at the moment. Neither Microsoft nor Google have resources dedicated to it. Many quirks and bugs remain unaddressed for a long time.

It's such a shame.
I haven't even found an exact point in time where we can run VR correctly.

I looked through the chromium community and couldn't find a plan for stable operation.
